apps/cookbook/src/app/examples/modal-example/modal-route-example/modal-route-page1-example.component.ts

```diff
@@ -8,8 +8,8 @@ import { ActivatedRoute } from '@angular/router';
     <h4>The standard Lorem Ipsum passage, used since the 1500s</h4>
     <span
       >QueryParams:
-      <pre>{{ queryParams$ | async | json }}</pre></span
-    >
+      <pre>{{ queryParams$ | async | json }}</pre>
+    </span>
```

Comment:
Wow. That was ugly before 🙂

But line 9-10 still look a bit odd. I would expect the `>` to be closing the `span` on line 9.